feature/qos/tests/qos_ecn_config_test/metadata.textproto

```diff
@@ -23,3 +23,11 @@ platform_exceptions: {
     qos_buffer_allocation_config_required: true
   }
 }
+platform_exceptions: {
+  platform: {
+    vendor: ARISTA
+  }
+  deviations: {
+    state_path_unsupported: true
+  }
+}
```

Comment:
Hi @sambhavjain-204 Can you add a new line here

Reply:
done